plugins/admission/Dockerfile

```diff
@@ -13,12 +13,14 @@ COPY opa opa
 WORKDIR /go/src/github.com/fairwindsops/insights-plugins/plugins/admission
 
 COPY ./admission/go.* ./
+COPY ./admission/version.txt .
+COPY ./admission/embed.go .
+COPY ./admission/pkg pkg
+COPY ./admission/cmd cmd
```

Comment:
I would keep the `COPY` statements below the `go mod download` for better caching.

Reply:
I've moved these back where they were. I had previously needed to copy some files before `go mod download` to avoid failure to find our new version pkg, but that is now solved by a `replace ...admission => ./` in the Admission go.mod so it doesn't look externally for the version pkg.